avalon/houdini/pipeline.py

```diff
@@ -10,8 +10,8 @@
 
 # Local libraries
 from . import lib
-from ..lib import logger
-from avalon import api, schema
+from ..lib import logger, find_submodule
+from avalon import api
```

Comment:
Shouldn't this be `from .. import api`?

Reply:
It sure should. I just removed the `schema` part but will fix it right up! Well spotted.

Reply:
Resolved with https://github.com/getavalon/core/pull/501/commits/986157351ac5bd6ea8dd5d22cb1ec4ac892952c3